tests/liteclient_test.cc

```diff
@@ -220,19 +250,17 @@ class LiteClientTest : public ::testing::Test {
                     tuf_repo_{test_dir_.Path() / "repo"},
                     ostree_{(test_dir_.Path() / "treehub").string(), true},
                     device_gateway_{ostree_, tuf_repo_},
-                    initial_target_{Uptane::Target::Unknown()} {
-
-    auto initial_sysroot_commit_hash = sysrepo_.repo().commit(sysrootfs_.path, sysrootfs_.branch);
-    sysrepo_.deploy(initial_sysroot_commit_hash);
-
-    Json::Value target_json;
-    target_json["hashes"]["sha256"] = initial_sysroot_commit_hash;
-    target_json["custom"]["targetFormat"] = "OSTREE";
-    target_json["length"] = 0;
-    initial_target_ = Uptane::Target{hw_id + "-" + os + "-1", target_json};
+                    initial_target_{Uptane::Target::Unknown()},
+                    sysroot_hash_{sysrepo_.repo().commit(sysrootfs_.path, sysrootfs_.branch)} {
+    sysrepo_.deploy(sysroot_hash_);
   }
 
-  std::shared_ptr<LiteClient> create_liteclient(bool initial_version = true) {
+  enum initial_version { kOff = 0, kOn = 1, kCorrupted1 = 2, kCorrupted2 = 3 };
```

Comment:
https://google.github.io/styleguide/cppguide.html#Type_Names (--> enum class InitialVersion).

Reply:
going to cleanup the whole file...is full of inconsistencies